src/rantanplan/structures.py

```diff
@@ -11,230 +10,155 @@
 # )
 # Structures will be checked in order of definition, the first one to match
 # will be chosen.
+ASSONANT_RHYME = "assonant"
+CONSONANT_RHYME = "consonant"
+ARTE_MAYOR_MIN_LENGTH = 9
+ARTE_MENOR_MAX_LENGTH = 8
+COPLA_ARTE_MENOR_MIN_LENGTH = 4
+LIRA_LONG_LINE = 11
+LIRA_SHORT_LINE = 7
+SILVA_LONG_LINE = LIRA_LONG_LINE
+SILVA_SHORT_LINE = LIRA_SHORT_LINE
+HENDECASYLLABLE = 11
+ALEXANDRINE = 14
+OCTOSYLLABLE = ARTE_MENOR_MAX_LENGTH
+TETRASYLLABLE = COPLA_ARTE_MENOR_MIN_LENGTH
+MINIMUM_SAFE_LENGTH = 3
+MAXIMUM_SAFE_LENGTH = 20
+#    STRUCTURES_LENGTH is a dictionary with a list of line lengths and a `True`
+#    or `False` flag to indicate if the structure can be cyclic or not
```

Comment:
Could be repeated? Still unsure about it, It might be worth asking around. And BTW, _mis dieses_ for the change.

Reply:
Sure thing bro